x-pack/plugins/cases/server/client/metrics/get_case_metrics.test.ts

```diff
@@ -85,8 +53,11 @@ describe('getMetrics', () => {
       clientArgs
     );
 
-    expect(metrics.alerts?.hosts).toBeDefined();
-    expect(metrics.alerts?.users).toBeDefined();
+    expect(metrics.alerts?.hosts).toEqual({
+      total: 2,
+      values: [{ name: 'host1', id: '1', count: 1 }],
+    });
+    expect(metrics.alerts?.users).toBeUndefined();
```

Comment:
Now that we're implementing the metrics since we only passed `alertHosts` we should receive anything regarding users.